src/oc_google_reporter_v2.erl

```diff
@@ -72,10 +72,15 @@ trunc_string(V) ->
     #{<<"value">> => V,
       <<"truncatedByteCount">> => 0}.
 
+format_attribute_map(_K, V) when is_integer(V) -> #{<<"int_value">> => V};
+format_attribute_map(_K, V) when is_boolean(V) -> #{<<"bool_value">> => V};
+format_attribute_map(_K, V) -> #{<<"string_value">> => trunc_string(V)}.
```

Comment:
Should we be doing anything to check that it's actually a string that has been passed in? You might be able to use the `is_bitstring/1` guard.

Reply:
I kind of want everything to fallback to this "formatter", an example might be an atom e.g. a phoenix action.

Reply:
What will happen in that case? I guess the JSON encoder will be able to handle some data types that might come through (e.g. atoms), but it might struggle with others or send an invalid trace.

Reply:
Currently the atoms are successfully sent as strings.